docs/rfcs/2025-04-14-signing-messages.md

```diff
@@ -0,0 +1,83 @@
+# Signing some member member messages and profiles
+
+## Problems
+
+1. Authenticity of profile addresses.
+
+When somebody sends a profile, e.g. when connecting via one time link, it may include contact address. It may be incorrectly used to claim that the sending person is the same person as one who published the same address in some trustworthy channel, like a social media account or a website. This claim is unproven, and can potentially be misleading - the address can be simply copy-pasted, together with other attributes of the profile. The solution to that would be to sign the profile or a message including the profile with the key that has its public part included in immutable address data. If only profile is signed, it can also be copy-pasted, but if the message is signed and if it includes some connection attributes (e.g. security code), then it would be impossible to copy paste. So the latter approach is preferred.
+
+Additional social media addresses can be included in the account, and they can also be validated via signature by the key from the short link published on social media account (requiring access to that account).
+
+With Nostr that uses secp256k1 key as identity it's even better, as ownership of this identity can be validated without contacting nostr, simply by verifying signature with Nostr public key.
+
+2. Authenticity of owners' and admins messages delivered via chat relays (aka superpeers).
+
+For the next generation of groups we want to protect from attacks by chat relays who otherwise could deliver messages that they have to right to send. E.g., messages that can only be sent by owners and or by the admin.
```

Comment:
```suggestion
For the next generation of groups we want to protect from attacks by chat relays who otherwise could deliver messages that they have no right to send. E.g., messages that can only be sent by owners and or by the admin.
```